### PR TITLE
Xlayer docs update zknodes

### PR DESCRIPTION
**Pull request by zjg555543**

### CODE REVIEW

File: developer/build-on-xlayer/zkevm-contracts.mdx [hunk 1]
```diff
@@ -1,38 +1,38 @@
 # zkEVM contract
-## X Layer L1 zkEVM contracts
+## X Layer zkEVM contracts
 These smart contracts facilitate zkEVM operation on Ethereum. 
 
-### Ethereum Layer 1 (Mainnet)
+### Ethereum (Mainnet, Layer 1)
```
Comment (zeshengOKX) [inline developer/build-on-xlayer/zkevm-contracts.mdx:5]: (Layer 1 mainnet)

File: developer/build-on-xlayer/zkevm-contracts.mdx [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+|[ZkEVMGlobalExitRoot.sol](https://github.com/okx/xlayer-contracts/blob/release/v0.3.0/contracts/v2/PolygonZkEVMGlobalExitRootV2.sol "ZkEVMGlobalExitRoot.sol")|Global exit root manager smart contract|[0x580bda1e7A0CFAe92Fa7F6c20A3794F169CE3CFb](https://etherscan.io/address/0x580bda1e7A0CFAe92Fa7F6c20A3794F169CE3CFb "0x580bda1e7A0CFAe92Fa7F6c20A3794F169CE3CFb")|
+|[DataCommittee.sol](https://github.com/okx/xlayer-contracts/blob/release/v0.3.0/contracts/v2/consensus/validium/PolygonDataCommittee.sol "DataCommittee.sol")|Data availability committees smart contract|[0x05652Ec92366F3C2255991a265c499E01Ba58e6a](https://etherscan.io/address/0x05652Ec92366F3C2255991a265c499E01Ba58e6a "0x05652Ec92366F3C2255991a265c499E01Ba58e6a")|
 |OKBContract.sol|OKB smart contract|[0x75231f58b43240c9718dd58b4967c5114342a86c](https://etherscan.io/address/0x75231f58b43240c9718dd58b4967c5114342a86c "0x75231f58b43240c9718dd58b4967c5114342a86c")|
 
-### Ethereum Layer 1 (Testnet)
+### X Layer (Mainnet, Layer 1)
```
Comment (zeshengOKX) [inline developer/build-on-xlayer/zkevm-contracts.mdx]: (Layer 1 mainnet)

File: developer/build-on-xlayer/zkevm-contracts.mdx [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
+|Name|Detail|Testnet address|
+|:----|:--|:--|
+|[ZkEVMBridge.sol](https://github.com/okx/xlayer-contracts/blob/v0.3.1/contracts/v2/PolygonZkEVMBridgeV2.sol "ZkEVMBridge.sol")|Bridge smart contract|[0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe](https://www.okx.com/explorer/xlayer/address/0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe "0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe")|
+|[ZkEVMGlobalExitRootL2.sol](https://github.com/okx/xlayer-contracts/blob/v0.3.1/contracts/v2/PolygonZkEVMGlobalExitRootV2.sol "ZkEVMGlobalExitRootL2.sol")|Global exit root manager smart contract|[0xa40d5f56745a118d0906a34e69aec8c0db1cb8fa](https://www.okx.com/explorer/xlayer/address/0xa40d5f56745a118d0906a34e69aec8c0db1cb8fa "0xa40d5f56745a118d0906a34e69aec8c0db1cb8fa")|
+|[WETH](https://www.okx.com/explorer/xlayer/token/0x5a77f1443d16ee5761d310e38b62f77f726bc71c "WETH")|WETH Token Addres| [0x5a77f1443d16ee5761d310e38b62f77f726bc71c](https://www.okx.com/explorer/xlayer/address/0x5a77f1443d16ee5761d310e38b62f77f726bc71c "0x5a77f1443d16ee5761d310e38b62f77f726bc71c")|
+
+### Ethereum Sepolia(Testnet, Layer 2)
```
Comment (zeshengOKX) [inline developer/build-on-xlayer/zkevm-contracts.mdx]: (Layer 2 testnet)

File: developer/build-on-xlayer/zkevm-contracts.mdx [hunk 4]
```diff
@@ -1,38 +1,38 @@
...
 |[ZkEVMGlobalExitRoot.sol](https://github.com/okx/xlayer-contracts/blob/release/v0.3.0/contracts/v2/PolygonZkEVMGlobalExitRootV2.sol "ZkEVMGlobalExitRoot.sol")|Global exit root manager smart contract|[0x66E61bA00F58b857A9DD2C500F3aBc424A46BD20](https://sepolia.etherscan.io/address/0x66E61bA00F58b857A9DD2C500F3aBc424A46BD20 "0x66E61bA00F58b857A9DD2C500F3aBc424A46BD20")|
 |[DataCommittee.sol](https://github.com/okx/xlayer-contracts/blob/release/v0.3.0/contracts/v2/consensus/validium/PolygonDataCommittee.sol "DataCommittee.sol")|Data availability committees smart contract|[0x9d89De09fa8666cD82fDb0E2466bB56D704D60e7](https://sepolia.etherscan.io/address/0x9d89De09fa8666cD82fDb0E2466bB56D704D60e7 "0x9d89De09fa8666cD82fDb0E2466bB56D704D60e7")|
 |OKBContract.sol|OKB smart contract|[0x3F4B6664338F23d2397c953f2AB4Ce8031663f80](https://sepolia.etherscan.io/address/0x3F4B6664338F23d2397c953f2AB4Ce8031663f80 "0x3F4B6664338F23d2397c953f2AB4Ce8031663f80")|
 
 
-### X Layer (Layer 2 Testnet)
+### X Layer (Testnet, Layer 2)
```
Comment (zeshengOKX) [inline developer/build-on-xlayer/zkevm-contracts.mdx:35]: (Layer 2 testnet)

File: developer/setup-zknode/setup-production-zknode.mdx [hunk 5]
```diff
@@ -23,7 +23,7 @@ Here is a list of crucial network components that are required before you can ru
 Let's set up each of the above components!
 
 ## Ethereum Node Setup
-The Ethereum RPC Node is the first component to be deployed because zkNode needs to synchronize blocks and transactions on L1. You can invoke the ETH RPC (Testnet: Sepolia) service through any of the following methods:
+The Ethereum RPC Node is the first component to be deployed because zkNode needs to synchronize blocks and transactions on L1. You can invoke the ETH RPC (Mainnet: Ethereum OR Testnet: Sepolia) service through any of the following methods:
```
Comment (zeshengOKX) [inline developer/setup-zknode/setup-production-zknode.mdx:26]: or

**Review by zeshengOKX — CHANGES_REQUESTED** [review]
内容小改动

**Review by sylbj23 — APPROVED** [review]

**Review by sylbj23 — APPROVED** [review]

**Review by sylbj23 — APPROVED** [review]

**Pull request merged**